src/services/email/libs/data-lib.js

```diff
@@ -0,0 +1,95 @@
+import { DateTime } from "luxon";
+
+import { getLookupValues } from "./lookup-lib";
+
+const formatAttachments = (formatType, attachmentList) => {
+    const formatChoices = {
+        "text": {
+            begin: "\n\n",
+            joiner: "\n",
+            end: "\n\n"
+        },
+        "html": {
+            begin: "<ul><li>",
+            joiner: "</li><li>",
+            end: "</li></ul>"
+        },
+    };
+    const format = formatChoices[formatType];
+    if (!format) {
+        console.log("new format type? ", formatType);
+        return "attachment List";
+    }
+    if (!attachmentList || attachmentList.length === 0)
+        return "no attachments";
+    else
+        return `${format.begin}${attachmentList.map(a => `${a.title}: ${a.filename}`).join(format.joiner)}${format.end}`;
+};
+
+const formatDateFromTimestamp = (timestamp) => {
+    if (!timestamp || timestamp <= 0) return "Pending";
+    return DateTime.fromMillis(timestamp)
+        .toFormat("DDDD");
+
+};
+
+function formatNinetyDaysDate(emailBundle) {
+    if (!emailBundle?.notificationMetadata?.submissionDate) return "Pending";
+    return DateTime.fromMillis(emailBundle.notificationMetadata.submissionDate)
+        .plus({ days: 90 })
+        .toFormat("DDDD '@ 11:59pm ET'");
+
+}
+
+export const buildEmailData = async (bundle, data) => {
+    const returnObject = {};
+
+    const lookupValues = await getLookupValues(bundle.lookupList, data.id);
+    data = { ...data, ...lookupValues };
```

Comment:
I dont personally love structuring new elements (lookupValues) back into the data property that was originally a method input.....but not the end of the world! :)

Reply:
Yeah, I was probably testing something with the lookups, trying to change as little as possible of the code further down and forgot to go back and "do it right" - should not be hard to update!